pkg/commands/git.go

```diff
@@ -21,6 +22,13 @@ import (
 	gogit "gopkg.in/src-d/go-git.v4"
 )
 
+// this takes something like:
```

Comment:
What is this comment

Reply:
it's referring to the regex below. I removed the newline inbetween to make the association clearer